attributes/defaults.rb

```diff
@@ -54,6 +60,14 @@
   default['mysqld']['my.cnf']['mysqld_safe']['skip_log_error'] = true
 end
 
+# Password columns for user passwords
+default['mysqld']['pwd_col'] = case node['mysqld']['db_install']
```

Comment:
An attribute won't hurt, and opens up the option for advanced users to use this cookbook with older mysql versions.
The only concern I have is, that the cookbook advises and deploys defaults that are unsafe, not sane, etc. I'm not sure howto "enforce" this (or whether it's a good idea to do so), besides a warning in the README.